<launch>
  <node pkg="trigger_cam" type="trigger_cam_node" name="trigger_cam" output="screen">
    <param name="device" value="/dev/video0"/>
    <param name="topic" value="image_raw"/>
    <param name="frame_id" value="usb_cam"/>

    <!-- Pick a mode your camera supports; for your device: 1280x720@15 or 1920x1080@8 in UYVY -->
    <param name="image_width" value="1920"/>
    <param name="image_height" value="1080"/>
    <param name="framerate" value="30"/>

    <!-- UYVY is safest in trigger mode -->
    <param name="pixel_format" value="MJPG"/>   <!-- or MJPG if you proved it's valid under trigger -->

    <!-- GStreamer path is more robust for UYVY -->
    <param name="use_gstreamer" value="false"/>
    <param name="buffer_size" value="1"/>
  </node>
</launch>
